gradle-sls-packaging/src/main/groovy/com/palantir/gradle/dist/BaseDistributionExtension.groovy

```diff
@@ -83,6 +94,14 @@ class BaseDistributionExtension {
         productDependencies.add(dependency)
     }
 
+    void setProductDependenciesConfig(Configuration productDependenciesConfig) {
+        this.productDependenciesConfig = productDependenciesConfig
+    }
+
+    void ignoredProductDependency(String productGroup, String productName) {
```

Comment:
@jared2501 could you mention this in your docs PR? I was pretty surprised a 2.4 -> 2.7 upgrade broke my build due to a pdep from MP!